src/utils/responseParser/index.ts

```diff
@@ -11,13 +13,13 @@ import {
 import type { GetTransactionReceiptResponse } from '../transactionReceipt';
 
 export abstract class ResponseParser {
-  abstract parseGetBlockResponse(res: any): GetBlockResponse;
+  abstract parseGetBlockResponse(res: BlockWithTxHashes): GetBlockResponse;
```

Comment:
I have a serious doubt for this one.
@tabaktoni  should have a look at all the types you added.